src/js/El.js

```diff
@@ -0,0 +1,146 @@
+import document from 'global/document';
+import window from 'global/window';
+import * as Dom from './utils/dom.js';
+import * as stylesheet from './utils/stylesheet.js';
+import {IE_VERSION} from './utils/browser';
+
+class El {
```

Comment:
Is El supposed to stand for language?

Reply:
Yeah. I think it's supposed to be a dom element, and improving the name could help clarify that.